c2rust-transpile/src/translator/simd.rs

```diff
@@ -428,16 +428,9 @@ impl<'c> Translation<'c> {
         let param_translation =
             self.convert_exprs(ctx.used(), &[first_expr_id, second_expr_id, mask_expr_id])?;
```

Comment:
This change would be obviated by [`array::try_map`](https://doc.rust-lang.org/std/primitive.array.html#method.map)/[`#![feature(array_try_map)]`](https://github.com/rust-lang/rust/issues/79711), but it doesn't look like it'll land soon.  [`array::map`](https://doc.rust-lang.org/std/primitive.array.html#method.map) is stable, but that doesn't work because of the error handling.

Reply:
What change? I didn't change anything here. If you mean that the `self.convert_exprs` call can be removed, that is problematic because of the `WithStmts` type, which cannot be constructed from an array via any current means (fixed-sized iterators are not even unstable).

Reply:
Sorry, I meant this comment more as a lament that this can't be written more nicely.  It's fine the way you wrote it, modulo my other comments.